build.sbt

```diff
@@ -83,7 +94,6 @@ lazy val vercors = (project in file("."))
     javacOptions in Compile += "-Xlint:deprecation",
     javacOptions in Compile += "-Xlint:unchecked",
     javacOptions in Compile += "-deprecation",
-    javacOptions in doc := Seq(),

```

Comment:
Does this work? I believe it inherits the options from Compile, hence the empty Seq

Reply:
After removing that line this is what I get in sbt:

```sbt
[IJ]show doc / javacOptions
[info] hre / doc / javacOptions
[info] 	List()
[info] col / doc / javacOptions
[info] 	List()
[info] viper_api / doc / javacOptions
[info] 	List()
[info] parsers / doc / javacOptions
[info] 	List()
[info] doc / javacOptions
[info] 	List()
```

So looks like it was changed in one of the SBT versions. At least the "stage" task and such still work.